src/client/components/pages/parts/privs-edit-modal.js

```diff
@@ -80,14 +83,47 @@ class PrivsEditModal extends React.Component {
 	handleBitChange(bit) {
 		const newPrivs = this.state.privs ^ (1 << bit);
 		if (this.props.targetUser.privs !== newPrivs) {
+			// If we have also added a note, then set submittable also true
+			if (this.state.note.length) {
+				this.setState({
+					privs: newPrivs,
+					submittable: true
+				});
+			}
+			else {
+				this.setState({
+					privs: newPrivs,
+					submittable: false
+				});
+			}
```

Comment:
```suggestion
			// If we have also added a note, then set submittable also true
			this.setState({
				privs: newPrivs,
				submittable: Boolean(this.state.note.length)
			});
```
Alternatively, if you are a fan of cryptic code, like me:
```suggestion
			// If we have also added a note, then set submittable also true
			this.setState({
				privs: newPrivs,
				submittable: !!this.state.note.length
			});
```
This simplification could even be extended to include the else-branch below, but GitHub doesn't allow me to make suggestions for deleted lines.